Formula/esbuild.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class Esbuild < Formula
+  desc "Extremely fast JavaScript bundler and minifier"
+  homepage "https://esbuild.github.io/"
+  url "https://registry.npmjs.org/esbuild/-/esbuild-0.12.21.tgz"
+  sha256 "d0c130384288f48361c826a6ac18af61eacd3db90ce0a89eaa575cf753b37e78"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    (testpath/"app.jsx").write <<~EOS
+      import * as React from 'react'
+      import * as Server from 'react-dom/server'
+
+      let Greet = () => <h1>Hello, world!</h1>
+      console.log(Server.renderToString(<Greet />))
+    EOS
+
+    system "npm", "install", "react", "react-dom", *Language::Node.std_npm_install_args(libexec)
```

Comment:
I really really doubt we want to install to the libexec dir here. Why not `testpath`?

Reply:
I really just need `npm install react react-dom` I dont think I really need `*Language::Node.std_npm_install_args` at all 😅 

We might need to update the audit check on the npm install stuff (it makes sense for build, but not for test)

Reply:
How does the `local_npm_install_args` behave?

Reply:
I actually dont think `local_npm_install_args` because it tries build from source as `std_npm_install_args`

Reply:
I guess the proper way would be to permit this in audit https://github.com/Homebrew/brew/blob/master/Library/Homebrew/rubocops/lines.rb#L517

Some audit-bypass hacks that may work:
- Use full path instead, which may bypass audit as the first argument doesn't match string "npm":
```suggestion
    system Formula["node"].opt_bin/"npm", "install", "react", "react-dom"
```
- Remove the build argument `Language::Node.local_npm_install_args.reject { |a| a["build-from-source"] }`

Reply:
good idea, thanks @cho-m!